### PR TITLE
Add OpenAPI Documentation pane

### PR DESCRIPTION
**Pull request by sbont**
To be published with 10.11.

### CODE REVIEW

File: content/en/docs/refguide/modeling/menus/view-menu/_index.md [hunk 1]
```diff
@@ -95,57 +95,61 @@ For details on using Mendix Marketplace components in your app modeling, see [Us
 
 For details on this intelligent virtual co-developer bot, see [MxAssist Best Practice Bot](/refguide/mx-assist-performance-bot/).
 
-### 3.11 Page Explorer
+### 3.11 OpenAPI Documentation
+
+The **OpenAPI Documentation** pane displays the auto-generated OpenAPI documentation for your currently opened Published OData service. This helps you to preview the REST operations that will be available once your app is published.
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/menus/view-menu/_index.md]: for _the_ currently opened Published OData service

File: content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/_index.md [hunk 2]
```diff
@@ -121,7 +121,9 @@ The $metadata XML file contains the service's contract in OData's [CSDL](https:/
 
 #### 3.2.3 OpenAPI {#openapi}
 
-The OpenAPI JSON file contains the service's REST contract in [OpenAPI 3.0](https://www.openapis.org/) format.
+The OpenAPI JSON file contains the service's REST contract in [OpenAPI 3.0](https://www.openapis.org/) format. This is a machine-readable file according to the OpenAPI Specification format. Most API tools support this format.
+
+When the app is running, this file can also downloaded from the [API documentation page](#api-documentation), under /odata-doc/{location}/openapi.json, where {location} is the location of the OData service (for instance, odata/myservice/v1).
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/_index.md]: Passive voice. How about "you can download this file"

File: content/en/docs/refguide/modeling/menus/view-menu/_index.md (hunk 1 above)
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/menus/view-menu/_index.md]: Make "Published OData service" a link to the documentation for published OData services?

File: content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/_index.md (hunk 2 above)
Comment (sbont) [reply content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/_index.md]: Yep, changed

File: content/en/docs/refguide/modeling/menus/view-menu/_index.md (hunk 1 above)
Comment (sbont) [reply content/en/docs/refguide/modeling/menus/view-menu/_index.md]: Good one, added

**Review by quinntracy — APPROVED** [review]

**Pull request merged**